### DISCUSSION TITLE
iOS 17 shows "Night" mode

### DISCUSSION THREAD

**Discussion by mjamison44** [homebridge-simplisafe3/homebridge-simplisafe3#402] [General]
After updating to iOS 17 the Simplisafe plugin is now showing "Night" as one of the modes. Prior to the update the only options were "Home", "Away", and "Off." 

Hopefully the developer can issue an update to fix this and remove the "Night" mode as it's not normally there and serves no purpose.

**Comment by shamoon** [comment]
Yea, its annoying.

It's unclear if this is a bug as in, Apple will fix it or a breaking change, as in they won't. We already do exclude the night option, it's just that it's being ignored. But apparently all devs are experiencing this ( https://github.com/home-assistant/core/issues/100898 )  and other device types still allow exclusion of modes, so hopefully (?) Apple will fix this.

If not, eventually, we'll have to do something lame like make the night button un-toggle itself to "home" or something

**Reply by devhayslett** [reply]
Seems as though it’s been fixed in 17.1, though now the Home App, at least on the phone, is showing a perpetual “Arming…” state whenever a change is made despite the base station recognizing and initiating the state change.

**Reply by shamoon** [reply]
Oh thats great news, Im not seeing an issue with the Arming thing, it's working normally over here. Maybe try giving things a restart etc.

**Comment by shamoon** [comment]
Closed by iOS 17.1 🎉
